services/core-web/src/components/noticeOfWork/applications/review/activities/Equipment.js

```diff
@@ -137,7 +137,7 @@ export const Equipment = (props) => {
         columns={columns(props.isViewMode)}
         dataSource={transformData(props.equipment)}
         locale={{
-          emptyText: "No equipment related to this activity",
+          emptyText: "No Data Yet",
```

Comment:
Should revert back to original description.

Reply:
Brian was looking for a consistent null screen message on all tables in the UI